### PR TITLE
[TVheadend] - New chart contribution

### PR DESCRIPTION
**Pull request by mr-onion-2**
New chart for [TVheadend](https://github.com/tvheadend/tvheadend), using the common library.

### CODE REVIEW

**Review by onedr0p — APPROVED** [review]

**Comment by onedr0p** [comment]
Thanks for another chart!

File: charts/tvheadend/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+appVersion: release-4.2
```
Comment (onedr0p) [inline charts/tvheadend/Chart.yaml:2]: This likely needs to be 4.2

**Review by onedr0p — CHANGES_REQUESTED** [review]

File: charts/tvheadend/Chart.yaml (hunk 1 above)
Comment (mr-onion-2) [reply charts/tvheadend/Chart.yaml:2]: Ah ok, I thought it should match the image tag

Comment (mr-onion-2) [reply charts/tvheadend/Chart.yaml:2]: The image version tag is indeed "release-4.2". Happy to change though, please confirm :)

Comment (onedr0p) [reply charts/tvheadend/Chart.yaml:2]: I think it should be the app version, not the image version. LSIO tagging is dumb

Comment (mr-onion-2) [reply charts/tvheadend/Chart.yaml]: Ahha! Thanks - learn something new every day

File: charts/tvheadend/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+appVersion: 4.2
```
Comment (onedr0p) [inline charts/tvheadend/Chart.yaml:2]: Actually looking at LSIO image tags for this image, I would roll with `appVersion: latest` here.

File: charts/tvheadend/values.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Default values for tvheadend.
+
+image:
+  repository: linuxserver/tvheadend
+  pullPolicy: IfNotPresent
+  tag: release-4.2
```
Comment (onedr0p) [inline charts/tvheadend/values.yaml:6]: Actually looking at LSIO image tags for this image, I would roll with `tag: latest` here.

**Review by onedr0p — CHANGES_REQUESTED** [review]
LSIO has the strangest way to tag, I would use `latest` on this. It is not the first chart we've had to do this with. Some applications developers and LSIO maintiners just don't follow proper semver. Usually I have been building our own images at https://github.com/k8s-at-home/container-images but this isn't even properly tagged by the developer hence LSIO dumb tags

**Comment by onedr0p** [comment]
Also worth nothing that v4.2 is very old but it looks like the developer is pretty active in commits in the repo.

File: charts/tvheadend/Chart.yaml (hunk 2 above)
Comment (mr-onion-2) [reply charts/tvheadend/Chart.yaml]: No probs! Done

File: charts/tvheadend/values.yaml (hunk 3 above)
Comment (mr-onion-2) [reply charts/tvheadend/values.yaml]: Done

**Comment by onedr0p** [comment]
Perfecto!

**Review by onedr0p — APPROVED** [review]

**Comment by mr-onion-2** [comment]
> Also worth nothing that v4.2 is very old but it looks like the developer is pretty active in commits in the repo. They just aren't tagging new releases.

Cheers yeh I noticed that. Waiting for [#1348](https://github.com/tvheadend/tvheadend/pull/1348) to be merged to allow Plex integration

**Comment by onedr0p** [comment]
So is this pretty much the same thing as xTeVe?

https://github.com/xteve-project/xTeVe

**Comment by mr-onion-2** [comment]
Similar yeh, as far I understand. Tvheadend supports physical tuners, and with tvhproxy the channels can be added to Plex DVR. The PR I referenced above will remove the need to use tvhproxy.

**Comment by Flole998** [comment]
> Also worth nothing that v4.2 is very old but it looks like the developer is pretty active in commits in the repo. They just aren't tagging new releases.

The problem is that the last active developer left right when the new release was supposed to happen soon, so I took over so the project isn't completely abandoned. I still need to find some time to figure out how a proper release is made (so the build system picks it up, official builds are created and so on) and that's why there is no release. Long story short: Tvheadend needs more active developers ;)

**Comment by onedr0p** [comment]
ThankS for the update @Flole998 ! We will be sure to update the chart once you release a new version :)

**Pull request merged**